scripts/pre_commit_hook.py

```diff
@@ -64,12 +64,18 @@ def install_hook():
     hooks_dir = os.path.join(oppia_dir, '.git', 'hooks')
     pre_commit_file = os.path.join(hooks_dir, 'pre-commit')
     chmod_cmd = ['chmod', '+x', pre_commit_file]
-    if os.path.islink(pre_commit_file):
+    if os.path.islink(pre_commit_file) and os.path.exists(pre_commit_file):
         python_utils.PRINT('Symlink already exists')
     else:
         # This is needed, because otherwise some systems symlink/copy the .pyc
         # file instead of the .py file.
         this_file = __file__.replace('pyc', 'py')
+        # If its a broken symlink, delete it.
+        is_link = os.path.islink(pre_commit_file)
```

Comment:
Again, too much duplication. Move these two definitions to above line 67, and then use them in line 67. 

Also, note that, in Oppia only methods/functions start with verbs. So, maybe change these to file_is_symlink and file_exists.

(Ditto for the other hook.)